### PR TITLE
Enable  TOC  and fix minor warnings

### LINKED ISSUE #614: Some error on the website
Cmd+Opt+U displays the following error

[Error] Refused to execute https://unimath.github.io/agda-unimath/theme/images/agda-unimath-logo.svg as script because image/svg+xml is not a script MIME type.

### PR DESCRIPTION
**Pull request by jonaprieto**
- Enhance TOC
- Solves some warnings seen here: https://github.com/UniMath/agda-unimath/actions/runs/4991616992/jobs/8939272325#step:12:1304
- Closes #614 
- Removes the background from the agda logo.

I tried syncing SVG stroke colours with agda-logo.css (so it responds to the colour theme), but no luck yet. We need more time to figure it out.  the following is related and the suggestion included in my experiment.

- https://github.com/rust-lang/mdBook/issues/773

### CODE REVIEW

**Comment by EgbertRijke** [comment]
Haha, our pre-commit is really strict with us these days:)

File: theme/images/agda-unimath-logo.svg
Comment (fredrik-bakke) [inline theme/images/agda-unimath-logo.svg:1]: Awesome! Are you also adding an inverted version for the darker themes?

Comment (jonaprieto) [reply theme/images/agda-unimath-logo.svg:1]: Yes, I'm working on it.

Comment (jonaprieto) [reply theme/images/agda-unimath-logo.svg:1]: Let a CSS controls the stroke colours.

**Pull request merged**